### ISSUE TITLE
Wordpress 3.6 causes error

### ISSUE THREAD

**Issue by crockett95** [romainberger/yeoman-wordpress#31]
Running OS X, Node v0.10.8, and `generator-yo-wordpress` 1.4.1. Error as follows:

```
-bash-3.2$ yo yo-wordpress


Trying to get the latest stable version of Wordpress

/usr/local/lib/node_modules/generator-yo-wordpress/node_modules/semver/semver.js:256
    throw new TypeError('Invalid Version: ' + version);
          ^
TypeError: Invalid Version: 3.6
    at new SemVer (/usr/local/lib/node_modules/generator-yo-wordpress/node_modules/semver/semver.js:256:11)
    at compare (/usr/local/lib/node_modules/generator-yo-wordpress/node_modules/semver/semver.js:409:10)
    at Function.gt (/usr/local/lib/node_modules/generator-yo-wordpress/node_modules/semver/semver.js:438:10)
    at /usr/local/lib/node_modules/generator-yo-wordpress/app/index.js:64:36
    at ChildProcess.exithandler (child_process.js:635:7)
    at ChildProcess.EventEmitter.emit (events.js:98:17)
    at maybeClose (child_process.js:735:16)
    at Socket.<anonymous> (child_process.js:948:11)
    at Socket.EventEmitter.emit (events.js:95:17)
    at Pipe.close (net.js:451:12)
```

Semver doesn't like Wordpress's version number 3.6.

**Comment by romainberger** [comment]
Damn. Didn't know semver was that strict. I'll push a fix as soon as I can

---

romainberger.com

**Comment by romainberger** [comment]
Fixed with #33
